src/ivoc/graph.cpp

```diff
@@ -1,14 +1,14 @@
 #include <../../nrnconf.h>
 
 #include <cstring>
-#include <string>
```

Comment:
If this include isn't needed, it's not needed, but this change seems unrelated to the purpose of the PR.